src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -830,20 +742,19 @@ void ProcessEvent(WifiPlatformEvent event)
  *       None
  **********************************************************************************/
 /* ARGSUSED */
-void sl_matter_wifi_task(void * arg)
+void MatterWifiTask(void * arg)
```

Comment:
should we keep the SL prefix in there?

Reply:
I don't think so - it is only used on the Silabs directories.